lib/Job.js

```diff
@@ -219,7 +220,10 @@ Job.prototype.schedule = function(spec) {
   } catch (err) {
     const type = typeof spec;
     if ((type === 'string') || (type === 'number')) {
-      spec = new Date(spec);
+      spec = rruleUtil.tryParseRRuleString(spec);
```

Comment:
This is not explicit enough. We need to add some kind of a mechanism to signal to the system here that what passed here is an RRule, we shouldn't be trying to parse everything as one.

Reply:
I partially agree. I think that there is no point attempting to parse it if RRule isnt included, but since the RRule is simply a string, there is no "good" way to determine if it is a valid RRule other than by attempting to parse it.

In which case ill add a function to check if the dependency is included and try to parse if it is.

Reply:
There are no reasons why it has to be just a string. One way to be more explicity is to introduce a new class, e. g. `class RRuleString { constructor (ruleString)}`, that user will be expected to pass when he wants to enable RRule functionality.